### ISSUE TITLE
[BERT/pytorch] hyper-parameter for Bert-Base

### ISSUE THREAD

**Issue by intersun** [NVIDIA/DeepLearningExamples#431]
Hi Teams,

I am trying to reproduce your results on **BERT-base** pre-training. We focus on BERT-base because our resource is limited and training BERT-large is too computation heavy.

So do you mind to share the hyper-parameters you used for BERT-base (if you ever tried to train bert-base), like what you presented for BERT-large? Especially what is the global batch size and learning rate and etc?   I tried to use the same parameter for bert-large, but it performs worse than bert-base released by google on GLUE tasks.

Thanks a lot for your help!

Siqi

**Comment by swethmandava** [comment]
We were able to train Bert Base with the same hyperparameters that were used for training Bert Large with LAMB optimizer and reproduce results on SQuAD. We have not tried GLUE but in the past observed that some of the GLUE scores (because of very small datasets) have high variance.

**Comment by sharathts** [comment]
See https://github.com/NVIDIA/DeepLearningExamples/issues/417
